src/fireblocks-sdk.ts

```diff
@@ -1605,15 +1611,14 @@ export class FireblocksSDK {
     }
 
     /**
-     *
-     * @param filter.pageCursor
-     * @param filter.pageSize
-     * @param filter.ids
-     * @param filter.sort
-     * @param filter.order
-     */
-    public async getNFTs(filter: GetNFTsFilter): Promise<Web3PagedResponse<Token>> {
-        const { pageCursor, pageSize, ids, sort, order } = filter;
+     * @param {Object} payload
+     * @param payload.pageCursor
+     * @param payload.pageSize
+     * @param payload.ids
+     * @param payload.sort
+     * @param payload.order
+     */
+    public async getNFTs({ pageCursor, pageSize, ids, sort, order }: GetNFTsFilter): Promise<Web3PagedResponse<Token>> {
```

Comment:
why here?

Reply:
keeping the same convention of using pagination filters

Reply:
it's just distructing on the signature level and not within the function. what's the convention that is kept?

Reply:
destruction in the signature 🙏🏻